lib/support/cli.js

```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
       describe: 'Choose which device to use. If you do not set it, random device will be used.',
       group: 'chrome'
     })
-    .option('chrome.dumpTraceCategoriesLog', {
+    .option('chrome.collectTraceCategoriesLog', {
       type: 'boolean',
-      describe: 'Dump Chromes traceCategories log to disk.',
+      describe: 'Collect Chromes traceCategories log to disk.',
```

Comment:
My understanding is that this means "include trace events in the perfLog". I suggest renaming the option and description to something that reflects that. Also, should this imply chrome.collectPerfLog? Maybe also chrome.traceCategories should imply this option (and by extension chrome.collectPerfLog).